Source/WebCore/style/StyleBuilder.h

```diff
@@ -35,7 +35,7 @@ namespace Style {
 class Builder {
     WTF_MAKE_FAST_ALLOCATED;
 public:
-    Builder(RenderStyle&, BuilderContext&&, const MatchResult&, CascadeLevel, PropertyCascade::IncludedProperties = PropertyCascade::IncludedProperties::All, const HashSet<AnimatableProperty>* = nullptr);
+    Builder(RenderStyle&, BuilderContext&&, const MatchResult&, CascadeLevel, PropertyCascade::IncludedProperties = PropertyCascade::IncludedProperties::All, const HashSet<AnimatableProperty>* animatedProperties = nullptr, HashSet<AnimatableProperty>* overriddenAnimatedProperties = nullptr);
```

Comment:
Passing overriddenAnimatedProperties in as a parameter everywhere seems bit unnecessary and fidly. Can it just be a member in the PropertyCascade? The client would access it after `styleBuilder.applyAllProperties()` via something like `styleBuilder. overriddenAnimatedProperties()`